process-engine/charon
# 🐛 Split interfaces

## What did you change?

This branch splits the interfaces a little bit further.

Closes #95 

## How can others test the changes?

This branch should have no effect on anything.
Just start charon with this branch and see if everything is functional.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
